src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -91,6 +92,7 @@
         <option value="BasicTestApp.TextOnlyComponent">Plain text</option>
         <option value="BasicTestApp.ToggleEventComponent">Toggle Event</option>
         <option value="BasicTestApp.TouchEventComponent">Touch events</option>
+        <option value="BasicTestApp.TwoWayBindingComponent">Two-way binding using delegate inside a loop.</option>
```

Comment:
If you're adding extra E2E scenarios, can you also add test code for them? For example in `src\Components\test\E2ETest\Tests\BindTest.cs` the behavior of `TwoWayBindingComponent` could be verified.

Reply:
That's a great idea. I'll do that.